logger/logger.config

```diff
@@ -1,10 +1,6 @@
-// This is the configuration file for the Runtime Logger.
-// Add comments by using the double slash in front of the line, like in C.
-// Do not change the order of the settings in the file, or the names of the settings.
-// Blank lines should not contain extra spaces, tabs, etc.
-// Yes/No answers can be input as any of (y/n), (yes/no), or capitalized.
-// Leave a space between the colon and the configuration setting.
-// Do not leave any setting blank.
+// This is the configuration file for the Runtime Logger, USE FOR TESTS IN TEST FRAMEWORK!!
+// DO NOT CHANGE!! If you change, you will most likely break a whole bunch of tests.
+// See logger/logger.config for an explanation for what all these fields mean
```

Comment:
Revert this file because it looks like the test config

Reply:
👌